db/utils.js

```diff
@@ -365,6 +413,8 @@ if (!Array.prototype.includes) {
 }
 
 module.exports = {
+  sortArrayAccordingToAnotherArray,
+  whereClauseForNameInArr,
```

Comment:
there is a db/helpers/common.js file for these kind of this. Can you move it there?

Reply:
ok will do